x/gov/tick.go

```diff
@@ -0,0 +1,144 @@
+package gov
+
+import (
+	"fmt"
+	"math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Called every block, process inflation, update validator set
+func EndBlocker(ctx sdk.Context, keeper Keeper) {
+
+	// Delete proposals that haven't met minDeposit
+
+	for shouldPopInactiveProposalQueue(ctx, keeper) {
+		inactiveProposal := keeper.InactiveProposalQueuePop(ctx)
+		if inactiveProposal.Status == "DepositPeriod" {
+			keeper.DeleteProposal(ctx, inactiveProposal)
```

Comment:
Log with tags when a proposal is deleted

Reply:
This isn't in a transaction.  Is there a way to add a tag here somehow?

Reply:
Tags can be returned in both `ResponseBeginBlock` and `ResponseEndBlock` (recent ABCI update)